multi-int-store/src/main/java/com/connexta/multiintstore/adaptors/S3StorageAdaptor.java

```diff
@@ -117,4 +117,9 @@ public RetrieveResponse retrieve(@NotBlank final String key) throws StorageExcep
       throw t;
     }
   }
+
+  @Override
+  public boolean objectExists(String key) {
+    return amazonS3.doesObjectExist(bucket, key);
```

Comment:
Why create this new method instead of throwing an exception in `#store` if the id already exists? We should never support overriding, so `#store` should fail for duplicate id regardless of whether `#objectExists` was checked first.